finalPrep/modZeroHero.js

```diff
@@ -56,29 +99,73 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
 
+var scaryMonster = {
+  name: "Fred",
+  smell: "Rotten Food",
+  weightInLbs: 1000,
+  citiesDestroyed: ["Tokyo","Paris"],
+  luckyNumbers: [7, 13],
+  address: {
+    number: 1122,
+    street: "Manhatten Blvd",
+    state: "Colorado",
+    zip: 80021,
+  }
+};
+console.log(scaryMonster);
 
 // Create a new class called SuperHero
 // - Your class should have the following DYNAMIC values
-//   - name 
+//   - name
 //   - superpower
-//   - age 
+//   - age
 // - Your class should have the following STATIC values
 //   - archNemesis, assigned to "The Syntax Error"
 //   - powerLevel = 100
-//   - energyLevel = 50 
+//   - energyLevel = 50
+class SuperHero {
+  constructor(name, superpower, age) {
+    this.name = name;
+    this.superpower = superpower;
+    this.age = age;
+    this.archNemesis = "Lex Luthor";
+    this.powerLevel = 100;
+    this.energyLevel = 50;
+  }
+  sayName(name) {
+    console.log(name);
+  }
+  maximizeEnergy() {
+    this.energyLevel = 1000;
+  }
+  gainPower(addPower) {
+    this.powerLevel = 100 + addPower;
+  }
+}
 
 // - Create the following class methods
 //   - sayName, should print the hero's name to the console
 //   - maximizeEnergy, should update the energyLevel to 1000
 //   - gainPower, should take an argument of a number and INCREASE the powerLevel by that number
 
 // - Create 2 instances of your SuperHero class
+var superHero1 = new SuperHero("William","Buying Beer", 30);
+var superHero2 = new SuperHero("Stephanie","Making Dinner", 26);
```

Comment:
✅ Convention would remind you to put spaced between the arguments you are passing